packages/react-components/src/organisms/__tests__/CheckboxGroup.test.tsx

```diff
@@ -18,6 +18,13 @@ it('renders a checkbox for each option', () => {
   expect(getByLabelText('Blue')).toBeVisible();
 });
 
+it('renders title', () => {
```

Comment:
Can we test this a little better. Im looking for a series of titles and options and that it comes out in the right order